src/SiteMaster/Core/Util.php

```diff
@@ -233,14 +233,23 @@ public static function getHTTPInfo($url, $options = array())
         
         curl_exec($curl);
 
-        $http_status = curl_getinfo($curl, CURLINFO_HTTP_CODE);
+        $http_status   = curl_getinfo($curl, CURLINFO_HTTP_CODE);
         $effective_url = curl_getinfo($curl, CURLINFO_EFFECTIVE_URL);
         $curl_error_no = curl_errno($curl);
 
         curl_close($curl);
         
+        //Always sleep after requesting data, this will help prevent loops from flooding a server.
+        sleep(1);
```

Comment:
Fixed in https://github.com/mfairchild365/site_master/commit/1fb69dd801c189e60214495f32b8742c71e7a81b